tests/frontend/test_kibana_app/cypress/integration/features/management/settings/logs.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Read Kibana logs
+
+  As a kibana user
+  I want to check the logs
```

Comment:
`I want to check the app logs` in order to differentiate from the cluster logs section

Reply:
Yes, that sounds much better